agent/engine/ordering_integ_test.go

```diff
@@ -339,6 +340,74 @@ func TestDependencyHealthyTimeout(t *testing.T) {
 	waitFinished(t, finished, orderingTimeout)
 }
 
+
+func TestGranularStopTimeout(t *testing.T) {
```

Comment:
This uses functions that don't exist in powershell. I'd move this to the _integ_linux_test.go file